client/src/i18n/index.js

```diff
@@ -118,7 +118,7 @@ const factory = (options = { debug: true }) =>
             },
             sessionList: {
               title: 'Your remembered sessions',
-              fullAccess: 'Full Access',
+              fullAccess: 'Edit',
```

Comment:
please rename 'fullAccess' to 'edit' (as you did previously) and remove 'fullAccess' from Polish dictionary

i18n is setup so that it will fall back to English translation, if it doesn't find Polish :)
if it doesn't find an English translation, it will show the key (e.g. `sessionList.fullAccess`), but it will always display _something_, so you don't have to worry about breaking stuff